### PR TITLE
Brand Template cleanup

### PR DESCRIPTION
**Pull request by sgaddameedi-insight**
# Description

Brand Template refine

Fixes # issue ticket number and link

## Type of change

1. Brand template

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] API unit tests
- [x] UI unit tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: UI/libs/core-admin/src/lib/components/brand/BrandForm.vue [hunk 1]
```diff
@@ -13,162 +13,276 @@
...
+                  <v-text-field
+                    :label="$t('Primary Theme Color')"
+                    :rules="[
+                      v => !!v || $t('Primary Theme color is required'),
+                      v =>
+                        (v && v.length <= 7) ||
+                        $t('Primary Theme must be less than 7 characters'),
+                      v =>
+                        (v && v.length > 0 && v.startsWith('#')) ||
+                        $t('Secondary Theme must start with #'),
```
Comment (jacobkellas) [inline UI/libs/core-admin/src/lib/components/brand/BrandForm.vue:132]: I think this should be "Primary"

**Review by jeremiahbooth-insight — APPROVED** [review]
Did I miss it or what are we displaying in that img tag if there is not an image.

File: UI/libs/core-admin/src/lib/components/brand/BrandForm.vue (hunk 1 above)
Comment (sgaddameedi-insight) [reply UI/libs/core-admin/src/lib/components/brand/BrandForm.vue:132]: Oops

**Comment by sgaddameedi-insight** [comment]
> Did I miss it or what are we displaying in that img tag if there is not an image.

Do you mean If there is no image in the settings API? I assume we will have a default logo for each app once we get the assets.

**Pull request merged**